pages/entity/_type/_.vue

```diff
@@ -135,15 +134,20 @@
           query: { page: 1 }
         }));
       }
+
       return axios.all([
         entities.getEntity(params.type, params.pathMatch, { wskey: env.EUROPEANA_ENTITY_API_KEY }),
         entities.relatedEntities(params.type, params.pathMatch, {
           wskey: env.EUROPEANA_API_KEY,
           entityKey: env.EUROPEANA_ENTITY_API_KEY
         }),
+        // TODO: DRY up (shared with search/index)
         search({
           page: currentPage,
-          query: `"${entities.getEntityUri(params.type, params.pathMatch)}"`,
+          qf: query.qf,
+          query: entityQuery,
```

Comment:
if a query is present in the url params we'll want to add that to the entityQuery.
Should they be 'AND'ed or do we need to use a fq like we do in the current portal for multiple search terms?

Reply:
That's outside of the scope of this piece of work. It is to follow for https://europeana.atlassian.net/browse/EC-3710.